lib/ansible/plugins/inventory/netbox.py

```diff
@@ -207,6 +209,12 @@ def extract_vcpus(self, host):
     def extract_memory(self, host):
         return host.get("memory")
 
+    def extract_platform(self, host):
+        try:
+            return self.platforms_lookup[host["platform"]["id"]]
+        except Exception:
```

Comment:
You can have ValueError (In case host[“platform"] is None), You can have KeyError in case platform exists but there is no id. Because I don't have the different treatments I don't see the point to specify more the exception.

Reply:
Ok, I understand, that's fine :)